include/libtorrent/kademlia/get_item.hpp

```diff
@@ -43,7 +43,7 @@ namespace libtorrent { namespace dht
 class get_item : public find_data
 {
 public:
-	typedef boost::function<bool(item&, bool)> data_callback;
+	typedef boost::function<void(item&, bool)> data_callback;
```

Comment:
we need this value just because we mixed put/get together to get. now we have nodes_callback used for put, get_items will always just try to get. so we needn't the return value anymore.